makefile.shade

```diff
@@ -50,18 +37,30 @@ var sqliteDllPath='src/Microsoft.Data.Sqlite/redist/x86/sqlite3.dll'
             {
                 CreatePartFromFile(
                     package,
-                    @"src/Microsoft.Data.Sqlite/build/net45/Microsoft.Data.Sqlite.targets",
-                    @"build/net45/Microsoft.Data.Sqlite.targets");
+                    sqliteDllPath,
+                    @"runtimes/win-x86/native/sqlite3.dll");
+                CreatePartFromFile(
+                    package,
+                    "src/Microsoft.Data.Sqlite/runtimes/win/native/x86/sqlite3.dll",
```

Comment:
It could, but doesn't need to. Per lines 21 & 22, they're both copies of the same thing.